### PR TITLE
Extending Facebook spec

### PR DESCRIPTION
**Pull request by AnSavvides**
I've added the following to the Facebook spec:
- More `optional` parameters to the `picture` path
- Removed an unnecessary parameter from the `search` path
- A `feed` path for publishing to a person's timeline
- Ability to like & comment on posts
- Ability to check in

Thanks for putting this cool project together!

### CODE REVIEW

File: specs/fb.json [hunk 1]
```diff
@@ -28,10 +32,53 @@
...
+                "coordinates": "optional",
+                "message": "optional",
+                "tags": "optional"
+            }
+        },
+        {
             "path": ":id/picture",
             "methods": [ "get" ],
             "params": {
-                "id": "required",
```
Comment (ttezel) [inline specs/fb.json:34]: why remove id here? I need to pull an `id` value from the params object to build the `:id/picture` url

**Comment by ttezel** [comment]
@AnSavvides awesome, thanks so much for the spec updates. Just had one comment above about the `:id/picture` params change.

File: specs/fb.json (hunk 1 above)
Comment (AnSavvides) [inline specs/fb.json:34]: But `id` does not come in as a parameter as far as I can see (http://developers.facebook.com/docs/reference/api/using-pictures/) - this is very similar to this where we have an `id` in the path but it is not listed in the `params`:

``` json
{
    "path": ":id",
    "methods": [ "get" ],
    "params": {
        "fields": "optional",
        "access_token": "required"
    }
}
```

Am I missing something?

Comment (ttezel) [inline specs/fb.json:34]: For example, in this url: `https://graph.facebook.com/shaverm/picture?redirect=false`, the id is "shaverm". But the id can also be a graph id, like 376995711728. So you would use it like so:

``` js
var params = {
  id: 'shaverm',
  access_token: 'FB-ACCESS-TOKEN',
  redirect: false
}

client
.use('fb')
 .get(':id/picture', params, function (err, res, reply) {
   console.log('reply', reply)
 })
```

Comment (AnSavvides) [inline specs/fb.json:34]: Ah, okay now this makes sense. Cool, no worries I shall put the `id` back to `params`.

**Comment by AnSavvides** [comment]
This should be good to go :+1:

**Comment by ttezel** [comment]
Hey @AnSavvides thanks a lot for the pull, the changes are up in unio v1.1.8, you can run `npm update unio` to get it :)

**Pull request merged**